webapp/components/test-file-results.html

```diff
@@ -205,21 +205,22 @@
       }
 
       async loadResultFile(testRun) {
-        const url = this.resultsURL(testRun);
+        const url = this.resultsURL(testRun, this.path);
         const response = await window.fetch(url);
         if (!response.ok) {
           return null;
         }
         return response.json();
       }
 
-      resultsURL(testRun) {
+      resultsURL(testRun, path) {
```

Comment:
Ahh, I didn't realize the parameters for `resultsURL` were different between `test-file-results` and `test-results-chart` in my #398 . Thanks!